source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/utils/wrappers/rsl_rl/vecenv_wrapper.py

```diff
@@ -173,6 +173,7 @@ def reset(self) -> tuple[torch.Tensor, dict]:  # noqa: D102
         return obs_dict["policy"], {"observations": obs_dict}
 
     def step(self, actions: torch.Tensor) -> tuple[torch.Tensor, torch.Tensor, torch.Tensor, dict]:
+        actions = actions.to(device=self.device)
```

Comment:
I assume this is a no-op when the tensor is already on the device?

Reply:
Exactly. This was necessary because before, training with 'cpu' simulation and 'cuda' PPO wasn't working before.

Reply:
@Dhoeller19 maybe we should push this to the environment itself instead of the wrapper?

Reply:
Done